docs/usage.rst

```diff
@@ -200,3 +200,99 @@ This is the case when you choose to integrate your user registration process wit
 Otherwise, serializer classes for ``verify`` and ``register`` views will not be available.
 
 3. Latest ``security_code`` generated for a ``phone_number`` can be found at ``/admin/phone_verify/smsverification/`` URL.
+
+
+Case 3: Create a user model and login the user once their phone number gets verified
+*************************************************************************************
+
+This case is suitable if you want to allow users to login to your django project using its amazing authentication backend.
+**Note**: The code below assumes you are using Django Templating system. The same logic would apply for a rest API.
+
+.. code-block:: python
+
+
+    # In models.py
+    from django.db import models
+    from django.contrib.auth.models import AbstractUser
+
+    # Create a custom user model by extending AbstractUser so that all the authetication functionality can be retained
+    class User(AbstractUser):
+        # Additional User fields can be added in here
+        phone_number = models.CharField(null=True, blank=True, max_length=20, unique=True)
+
+
```

Comment:
No need for so many empty lines here.
Please follow PEP8 and all code examples should be within 79 characters in general. We can make it 120, but ensure that the overall styling of code-blocks is consistent.

Reply:
Ok. Please see my new commit